aiperf/exporters/gpu_telemetry_console_exporter.py

```diff
@@ -0,0 +1,232 @@
+                    endpoints_successful = telemetry_results.endpoints_successful
+                    total_count = len(endpoints_tested)
+                    successful_count = len(endpoints_successful)
+                    failed_count = total_count - successful_count
+
+                    if failed_count == 0:
+                        title_lines.append(
+                            f"[bold green]{successful_count}/{total_count} DCGM endpoints reachable[/bold green]"
+                        )
+                    elif successful_count == 0:
+                        title_lines.append(
+                            f"[bold red]{successful_count}/{total_count} DCGM endpoints reachable[/bold red]"
+                        )
+                    else:
+                        title_lines.append(
+                            f"[bold yellow]{successful_count}/{total_count} DCGM endpoints reachable[/bold yellow]"
+                        )
+
+                    for endpoint in endpoints_tested:
+                        clean_endpoint = normalize_endpoint_display(endpoint)
+                        if endpoint in endpoints_successful:
+                            title_lines.append(f"[green]• {clean_endpoint} ✅[/green]")
+                        else:
+                            title_lines.append(
+                                f"[red]• {clean_endpoint} ❌ (unreachable)[/red]"
+                            )
+
+                    title_lines.append("")
+                    title_lines.append(table_title_base)
+                    table_title = "\n".join(title_lines)
+                else:
+                    renderables.append(Text(""))
+                    table_title = table_title_base
+
+                metrics_table = Table(
+                    show_header=True, title=table_title, title_style="italic"
+                )
+                metrics_table.add_column("Metric", justify="right", style="cyan")
+                metrics_table.add_column("avg", justify="right", style="green")
+                metrics_table.add_column("min", justify="right", style="green")
+                metrics_table.add_column("max", justify="right", style="green")
+                metrics_table.add_column("p99", justify="right", style="green")
+                metrics_table.add_column("p90", justify="right", style="green")
+                metrics_table.add_column("p75", justify="right", style="green")
+                metrics_table.add_column("std", justify="right", style="green")
+
+                metrics_to_display = [
+                    ("Power Usage (W)", "gpu_power_usage", "W"),
+                    ("Energy Consumption (MJ)", "energy_consumption", "MJ"),
+                    ("Utilization (%)", "gpu_utilization", "%"),
+                    ("Memory Used (GB)", "gpu_memory_used", "GB"),
+                    ("Temperature (°C)", "gpu_temperature", "°C"),
+                    ("SM Clock (MHz)", "sm_clock_frequency", "MHz"),
+                ]
```

Comment:
move this to the top of the file as a constant. Also, is this the same as the one for csv export?